packages/zambdas/src/ehr/submit-lab-order/index.ts

```diff
@@ -49,18 +50,16 @@ export const index = async (input: ZambdaInput): Promise<APIGatewayProxyResult>
     const locationID = serviceRequest.locationReference?.[0].reference?.replace('Location/', '');
 
     if (!appointment.id) {
-      throw new Error('appointment id is undefined');
+      throw EXTERNAL_LAB_ERROR('appointment id is undefined');
     }
-
     if (!encounter.id) {
-      throw new Error('encounter id is undefined');
+      throw EXTERNAL_LAB_ERROR('encounter id is undefined');
     }
     if (!serviceRequest.reasonCode) {
-      throw new Error('service request reasonCode is undefined');
+      throw EXTERNAL_LAB_ERROR('service request reasonCode is undefined');
     }
-
     if (!patient.id) {
-      throw new Error('patient.id is undefined');
+      throw EXTERNAL_LAB_ERROR('patient.id is undefined');
```

Comment:
Just for consistency with the other id type errors
```suggestion
      throw EXTERNAL_LAB_ERROR('patient id is undefined');
```